### PR TITLE
Refactor download url

### PR DESCRIPTION
**Pull request by brendarearden**

### CODE REVIEW

File: install-binary.sh [hunk 1]
```diff
@@ -1,95 +1,16 @@
...
-}
-
 # downloadFile downloads the latest binary package and also the checksum
 # for that binary.
+PROJECT_NAME="helm-unittest"
+
 downloadFile() {
-  PLUGIN_TMP_FILE="/tmp/${PROJECT_NAME}.tgz"
+  # Always use version from plugin.yaml for the download url
+  DOWNLOAD_URL="https://github.com/rancher/helm-unittest/releases/download/$CATTLE_HELM_UNITTEST_VERSION/$PROJECT_NAME-linux-$ARCH.tgz"
```
Comment (cbron) [inline install-binary.sh:10]: Looks like arch is set on line 25 above but that has been deleted ? Where does `CATTLE_HELM_UNITTEST_VERSION` come from ?

File: install-binary.sh [hunk 2]
```diff
@@ -1,95 +1,16 @@
...
-  if ! echo "${supported}" | grep -q "${OS}-${ARCH}"; then
-    echo "No prebuild binary for ${OS}-${ARCH}."
-    exit 1
-  fi
-
-  if ! type "curl" >/dev/null 2>&1 && ! type "wget" >/dev/null 2>&1; then
-    echo "Either curl or wget is required"
-    exit 1
-  fi
-}
```
Comment (cbron) [inline install-binary.sh:64]: Might leave this in here, as it won't hurt anything and will help in future if we support arm.

File: install-binary.sh [hunk 3]
```diff
@@ -99,10 +20,10 @@ installFile() {
   mkdir -p "$HELM_TMP"
   tar xf "$PLUGIN_TMP_FILE" -C "$HELM_TMP"
   HELM_TMP_BIN="$HELM_TMP/untt"
-  echo "Preparing to install into ${HELM_PLUGIN_PATH}"
+  echo "Preparing to install into $HELM_PLUGIN_DIR"
```
Comment (cbron) [inline install-binary.sh:23]: Where does `HELM_PLUGIN_DIR` get set ?

File: install-binary.sh [hunk 4]
```diff
@@ -1,95 +1,16 @@
...
-
-# Convert the HELM_PLUGIN_PATH to unix if cygpath is
-# available. This is the case when using MSYS2 or Cygwin
-# on Windows where helm returns a Windows path but we
-# need a Unix path
-if type cygpath &> /dev/null; then
-  HELM_PLUGIN_PATH=$(cygpath -u $HELM_PLUGIN_PATH)
-fi
-
-if [[ $SKIP_BIN_INSTALL == "1" ]]; then
```
Comment (cbron) [inline install-binary.sh:18]: No harm in keeping this, unless there is real reason for deleting.

**Review by cbron — CHANGES_REQUESTED** [review]

**Review by dramich — APPROVED** [review]

**Pull request merged**